holoviews/plotting/bokeh/links.py

```diff
@@ -94,31 +95,37 @@ def find_links(cls, root_plot):
                     # If link has no target don't look further
                     found.append((link, plot, None))
                     continue
-                potentials = [cls.find_link(p, link) for p in plots]
+                potentials = [cls.find_link(p, link, target=True) for p in plots]
                 tgt_links = [p for p in potentials if p is not None]
                 if tgt_links:
                     found.append((link, plot, tgt_links[0][0]))
         return found
 
     @classmethod
-    def find_link(cls, plot, link=None):
+    def find_link(cls, plot, link=None, target=False):
         """
         Searches a GenericElementPlot for a Link.
```

Comment:
Maybe add a small comment about `target` in the docstring.

Reply:
Done.